modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,287 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-network-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network, `br-ex1`, that has a different configuration to the default `br-ex` bridge. You must define the additional bridge in an interface definition of a machine configuration manifest file. The Machine Config Operator uses the manifest to create a new file at `/etc/ovnk/extra_bridge` on the host. The new file includes the name of the bond interface that hosts the additional `br-ex` bridge network.
+
+After you create and edit the manifest file, the Machine Config Operator completes tasks in the following order:
+
+. Drains nodes in singular order based on the selected machine configuration pool.
+. Injects Ignition configuration files into each node, so that each node receives the additional `br-ex` bridge network configuration.
+//. Restarts the node with the modified override file. Remove when content is SME approved.
+. Verify that the `br-ex` MAC address matches the MAC address for the interface that `br-ex` uses for the network connection.
+. Executes the `configure-ovs.sh` shell script that references the new interface definition.
+. Adds `br-ex` and `br-ex1` to the host node.
+. Uncordons the nodes. After all the nodes return to the `Ready` state, the Operator applies the `k8s.ovn.org/l3-gateway-config` annotation to each node.
```

Comment:
OVN-Kubernetes will apply the l3-gateway-config annotation after detecting and configuring br-ex and br-ex1 bridge.

Reply:
:heavy_check_mark: